server/lib/nl/fulfillment/time_delta_across_places.py

```diff
@@ -88,26 +89,36 @@ def _populate_cb(state: PopulateState, chart_vars: ChartVars,
   dcid2place = {c.dcid: c for c in child_places}
   dcids = list(dcid2place.keys())
 
-  ranked_child_dcids = utils.rank_places_by_growth_rate(
+  direction = state.time_delta_types[0]
+  ranked_children = utils.rank_places_by_series_growth(
       places=dcids,
       sv=chart_vars.svs[0],
-      growth_direction=state.time_delta_types[0],
+      growth_direction=direction,
       rank_order=rank_order)
 
-  utils.update_counter(state.uttr.counters, 'time-delta_reranked_places', {
-      'orig': dcids,
-      'ranked': ranked_child_dcids,
-  })
-  ranked_child_places = []
-  for d in ranked_child_dcids:
-    ranked_child_places.append(dcid2place[d])
+  utils.update_counter(
+      state.uttr.counters, 'time-delta_reranked_places', {
+          'orig': dcids,
+          'ranked_abs': ranked_children.abs,
+          'ranked_pct': ranked_children.pct,
+      })
+  block_id = chart_vars.block_id + 10
```

Comment:
curious why we need to add 10 for the block_id?

Reply:
Mainly to avoid some other chart from reusing the same block_id.  Needed only for the 2nd block being introduced here.